parse.go

```diff
@@ -278,29 +278,34 @@ func (lp *parser) parseTransaction(dateString, payeeString, payeeComment string)
 // with the remaining balance.
 func balanceTransaction(input *Transaction) error {
 	balance := decimal.Zero
-	var emptyFound bool
+	var numEmpty int
 	var emptyAccIndex int
 	if len(input.AccountChanges) < 2 {
 		return fmt.Errorf("need at least two postings")
 	}
 	for accIndex, accChange := range input.AccountChanges {
 		if accChange.Balance.IsZero() {
-			if emptyFound {
-				return fmt.Errorf("more than one account empty")
-			}
+			numEmpty++
 			emptyAccIndex = accIndex
-			emptyFound = true
 		} else {
 			balance = balance.Add(accChange.Balance)
 		}
 	}
-	if balance.Sign() != 0 {
-		if !emptyFound {
-			return fmt.Errorf("no empty account to place extra balance")
-		}
+
+	if balance.IsZero() {
+		// If everything adds up, then all is well.
```

Comment:
This is the key assumption - I don't think this will cause other issues but I'm happy to be corrected.